IPython/lib/pretty.py

```diff
@@ -484,10 +484,7 @@ def remove(self, group):
         except ValueError:
             pass
 
-try:
-    _baseclass_reprs = (object.__repr__, types.InstanceType.__repr__)
-except AttributeError:  # Python 3
-    _baseclass_reprs = (object.__repr__,)
```

Comment:
As it's a private variable, we could get rid of it entirely and just check `is not object.__repr__` below.